src/providers/fault-summary/cat-home-test/__tests__/fault-summary.cat-home-test.spec.ts

```diff
@@ -0,0 +1,319 @@
+import { TestBed } from '@angular/core/testing';
+
+import { configureTestSuite } from 'ng-bullet';
+import { FaultSummaryCatHomeTestHelper } from '../fault-summary.cat-home-test';
+import {
+    catFTestDataStateObject,
+    catFTestDataVCStateObject,
+} from '../../../fault-count/__mocks__/cat-F-test-data-state-object';
+import {
+    catGTestDataStateObject,
+    catGTestDataVCStateObject,
+} from '../../../fault-count/__mocks__/cat-G-test-data-state-object';
+import {
+    catHTestDataStateObject,
+    catHTestDataVCStateObject,
+} from '../../../fault-count/__mocks__/cat-H-test-data-state-object';
+import {
+    catKTestDataStateObject,
+    catKTestDataVCStateObject,
+} from '../../../fault-count/__mocks__/cat-K-test-data-state-object';
+
+describe('FaultSummaryCatHomeTestHelper', () => {
+
+  configureTestSuite(() => {
+    TestBed.configureTestingModule({
+    });
+  });
+
+  describe('getDrivingFaultsCatHomeTest (No VehicleChecks)', () => {
+    it(`should return 3 driving fault count for category F`, () => {
```

Comment:
No need to use back ticks on this test description and those below

Reply:
Copy and paste when I was using test cases that required value substitution, removed the substitution but didn't remove back ticks. Great spot.